packages/ckeditor5-utils/tests/collection.js

```diff
@@ -148,8 +148,31 @@ describe( 'Collection', () => {
 	} );
 
 	describe( 'add()', () => {
-		it( 'should be chainable', () => {
-			expect( collection.add( {} ) ).to.equal( collection );
+		it( 'should proxy its calls to addMany', () => {
+			const spy = sinon.spy( collection, 'addMany' );
+			const item = {};
+
+			collection.add( item );
+
+			sinon.assert.calledOnce( spy );
+			sinon.assert.calledWithExactly( spy, sinon.match.array.contains( [ item ] ), undefined );
+		} );
+
+		it( 'should proxy custom index', () => {
```

Comment:
The same comment as for previous test - I'd rather see those tests as the expected outcome and don't bother how the `Collection` do internally.

It would be much easier to refactor it later if needed).

Reply:
Reverted all the old tests in 8cd7ff93.

My intention here was to keep it DRY and avoid same tests in two methods.